<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovieDetailsDialog</class>
 <widget class="QDialog" name="MovieDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Информация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="posterLabel">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>200</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Тип:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeCombo"/>
       </item>
       <item>
        <widget class="QLineEdit" name="titleEdit">
         <property name="placeholderText">
          <string>Название фильма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="overviewEdit">
         <property name="placeholderText">
          <string>Описание фильма</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Жанр:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="genreCombo"/>
       </item>
       <item>
        <widget class="QLabel" name="ratingLabel">
         <property name="text">
          <string>Рейтинг: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ratingComboBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Год:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yearSpin">
         <property name="minimum">
          <number>1895</number>
         </property>
         <property name="maximum">
          <number>2100</number>
         </property>
         <property name="value">
          <number>2000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="directorEdit">
         <property name="placeholderText">
          <string>Режиссёр</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progressLabel">
         <property name="text">
          <string>Прогресс:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonsLayout">
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string>Сохранить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="text">
            <string>Закрыть</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
